docs/dpdk/gnmi-ctl.rst

```diff
@@ -0,0 +1,281 @@
+gnmi-ctl executable
+-------------------
+
+gnmi-ctl is a gRPC-based C++ network management interface client to handle port configurations and program fixed functions in the P4 pipeline. This client connects to the Config monitoring service running as part of Stratum and integrated as part of infrap4d. This utility is built as part of networking recipe and is available under `install/bin/`.
+
```

Comment:
I believe you need to use doubled tic marks in rst files. There are a number of places in the document that have single tics, and they aren't displayed in `code` font.

This is the second time I've posted this comment. It looks as if force-pushing your change erased the first one (!)

Reply:
Addressed

Reply:
Edited the document to have double tic marks